app/cli/README.md

```diff
@@ -57,6 +57,19 @@ We leverage buf.io to lint and generate proto files. Make sure you [install buf]
 make api
 ```
 
+### Updating default values
+
+By default CLI uses the following values for the Control Plane and Artifacts CAS API endpoints:
+
+- Artifacts CAS: api.cas.chainloop.dev:443
```

Comment:
Note for another Pull request, to make it clear,  I'd indicate that this is optional, that these values can be overridden "at runtime" on a per-need basis. https://github.com/chainloop-dev/chainloop#configure-cli-optional